.travis.yml

```diff
@@ -12,10 +12,14 @@ install:
   - true
 
 script:
+  - make validate
+  - make test-unit
+  
+  # make test-cmd requires kompose binary
   - make binary
+  
   # $GOPATH/bin is in $PATH
   - mkdir -p $GOPATH/bin
   - cp kompose $GOPATH/bin/
```

Comment:
a generic question: rather than doing `make binary` and moving it around why not use `make install` ?